### PR TITLE
docs: updated feature requests link in contributing file

### PR DESCRIPTION
**Pull request by mdfaizanahmed786**
### What does it do?
**It redirects to the updated link of Feature Requests.**

### Why it is needed?
**The old link was not redirecting to the correct page of Feature Requests.**

I have updated the link in Contributing file.

### CODE REVIEW

**Review by derrickmehaffy — DISMISSED** [review]
Lgtm

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ We highly appreciate your effort to contribute, but we recommend you talk to a m
 
 ## Feature Requests
 
-Feature Requests by the community are highly encouraged. Feel free to submit a new one or upvote an existing feature request on [ProductBoard](https://portal.productboard.com/strapi).
+Feature Requests by the community are highly encouraged. Feel free to submit a new one or upvote an existing feature request on [ProductBoard](https://feedback.strapi.io/feature-requests).
```
Comment (mdfaizanahmed786) [inline CONTRIBUTING.md:9]: I have added an updated link to the feature request.

**Review by derrickmehaffy — APPROVED** [review]
LGTM

**Comment by derrickmehaffy** [comment]
Please stop merging branches manually, everytime you do I have to retrigger our tests.

I can and will merge the branch manually when I'm ready to merge the PR.

**Pull request closed without merging**